homeassistant/components/octoprint/button.py

```diff
@@ -38,17 +40,20 @@ class OctoprintButton(CoordinatorEntity[OctoprintDataUpdateCoordinator], ButtonE
     """Represent an OctoPrint binary sensor."""
```

Comment:
Not introduced in this PR, but seems like a good time to fix it.

```suggestion
    """Represent an OctoPrint button."""
```